### PR TITLE
2447 assertJ haming

### PR DESCRIPTION
**Pull request by jennylia**
# pull request

Adding assertJ style to HammingTest
related to https://github.com/exercism/java/issues/2147

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/main/POLICIES.md#event-checklist)

### CODE REVIEW

**Comment by jennylia** [comment]
how do you add the hacktoberfest label like this PR did? https://github.com/exercism/java/pull/2172

**Comment by kerolloz** [comment]
> how do you add the hacktoberfest label like this PR did? #2172

A maintainer will add the label after approving and merging your changes. 
@jennylia

**Review by kerolloz — APPROVED** [review]

**Review by andrerfcsantos — APPROVED** [review]
Thanks!

**Pull request merged**
